atd-vze/package.json

```diff
@@ -49,8 +49,7 @@
     "react-csv-reader": "^3.0.6",
     "react-datepicker": "^2.9.6",
     "react-dom": "^16.8.6",
-    "react-map-gl": "^5.1.1",
-    "react-map-gl-draw": "^0.15.0",
```

Comment:
This package was only used by the unneeded `LocationEditMap` component removed in this PR. It isn't compatible with the new `react-map-gl` version anyways.

Reply:
✌️